File: Beginner/1048 - Salary Increase/1048.c
#include <stdio.h>

int main()
{

    double salario;
    int aux;

    scanf("%lf", &salario);

    if (salario <= 400.00)
        aux = 15;
    else if (salario <= 800.00)
        aux = 12;
    else if (salario <= 1200.00)
        aux = 10;
    else if (salario <= 2000.00)
        aux = 7;
    else
        aux = 4;

    printf("Novo salario: %.2lf\n", salario * (1 + aux / 100.00));
    printf("Reajuste ganho: %.2lf\n", salario * aux / 100.00);
    printf("Em percentual: %d \%\n", aux);

    return 0;
}

File: Beginner/1048 - Salary Increase/1048.cs
using System;

class URI
{
    static void Main(string[] args)
    {

        double input = double.Parse(Console.ReadLine());
        double output;
        int increase;

        if (input <= 400)
            increase = 15;
        else if (input <= 800)
            increase = 12;
        else if (input <= 1200)
            increase = 10;
        else if (input <= 2000)
            increase = 7;
        else
            increase = 4;
        double salary = input * (1 + (increase / 100));
        Console.WriteLine($"Novo salario: {salary:0.00}");
        Console.WriteLine($"Reajuste ganho: {salary - input:0.00}");
        Console.WriteLine($"Em percentual: {increase} %");
    }
}